sources/api/pluto/src/proxy.rs

```diff
@@ -1,5 +1,6 @@
+use crate::hyper_proxy::{Proxy, ProxyConnector};
+
 use hyper::Uri;
```

Comment:
nit: I prefer to keep these together and let rustfmt sort it. Not sure if we have an established style on this though.

```suggestion
use crate::hyper_proxy::{Proxy, ProxyConnector};
use hyper::Uri;
```